### PR TITLE
Flush DNS cache after connecting on MacOS clients

### LINKED ISSUE #1118: Consider option for flushing DNS cache after telepresence is active
Every time after running `telepresence`, I get a few more host entries available for access. E.g.: `http://remote-server:5050`.
 
But in macOS, due to DNS caching, I have to manually run `sudo killall -HUP mDNSResponder` every time after `telepresence` is loaded for it to actually start seeing the new entries.

My proposal is a new flag for `telepresence` that detects the OS and runs a dns-cache-killing command automatically.

Telepresence already has `sudo`, which probably makes this more acceptable.

### PR DESCRIPTION
**Pull request by KarolisL**
Fixes #1118

If there is a better way to implement this, please point me to it, I would be more than happy to implement it.

### CODE REVIEW

**Comment by KarolisL** [comment]
Thank you for review
! 
As I understand, I don't need to copy ENV vars to my `sudo pkill` process (as an arg to `check_call`), do I?

**Comment by ark3** [comment]
You are correct, you should not need environment variables for that.

File: telepresence/outbound/local.py [hunk 1]
```diff
@@ -137,5 +137,8 @@ def launch_vpn(
 def _flush_dns_cache(runner: Runner):
     if runner.platform == "darwin":
         runner.show("Connected. Flushing DNS cache.")
-        pkill_cmd = ["sudo", "-n", "/usr/bin/pkill", "-HUP", "mDNSResponder"]
-        runner.check_call(pkill_cmd)
+        pkill_cmd = ["sudo", "-n", "/usr/bin/pkill", "-HUP", "mDNSResponder_Z"]
```
Comment (ark3) [inline telepresence/outbound/local.py]: Why the `_Z` at the end?

Comment (KarolisL) [reply telepresence/outbound/local.py]: Sorry about that, I've used it to test error condition. I've removed it.

**Comment by ark3** [comment]
Thank you again for the PR.

**Comment by KarolisL** [comment]
Thank you for great tool! :-)

**Pull request merged**